### PR TITLE
Update mysql_4byte_support.rst

### PR DESCRIPTION
**Pull request by timbabcock**
The innodb_file_format command has been depreciated since MySQL, version 8.0 was released.

### CODE REVIEW

File: admin_manual/configuration_database/mysql_4byte_support.rst [hunk 1]
```diff
@@ -3,7 +3,8 @@ Enabling MySQL 4-byte support
 =============================
 
 .. note::
-
+    Changing of innobd_file_format was depreciated in MySQL Version 8.0 (see dev.mysql.com (Server and Status Variables and Options added, depreciated, or removed in MySqL 8.0).
```
Comment (MorrisJobke) [inline admin_manual/configuration_database/mysql_4byte_support.rst:6]: In MariaDB 10.2.2 Barracuda is the default and in 10.3 it can't be changed anymore: https://mariadb.com/kb/en/innodb-file-format/

The MySQL deprecation page: https://dev.mysql.com/doc/refman/8.0/en/added-deprecated-removed.html

**Comment by MorrisJobke** [comment]
Thanks for the hint. Let me rephrase the documentation steps directly to reflect this, because step 2 already contains this information.

**Comment by MorrisJobke** [comment]
> Thanks for the hint. Let me rephrase the documentation steps directly to reflect this, because step 2 already contains this information.

#5264

**Comment by MorrisJobke** [comment]
Is this okay for you?

**Comment by MorrisJobke** [comment]
Let me close this in favor of #5264

File: admin_manual/configuration_database/mysql_4byte_support.rst (hunk 1 above)
Comment (timbabcock) [reply admin_manual/configuration_database/mysql_4byte_support.rst:6]: This maybe a bit misleading:

Comment (timbabcock) [reply admin_manual/configuration_database/mysql_4byte_support.rst:6]: Sorry my response was late.  That change should do the trick.

Comment (MorrisJobke) [reply admin_manual/configuration_database/mysql_4byte_support.rst:6]: Is #5264 fine or does it need further refinements in your opinion?
